### PR TITLE
DDP-4052 Osteo Instagram Feed (Fixed branch!)

### PR DESCRIPTION
**Pull request by zyme**
Replacing the faulty transferred branch `DDP-4052_OsteoInstagramFeed`
Picking up on PR from old repo that was here: https://github.com/broadinstitute/ddp/pull/1225

### CODE REVIEW

File: ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.ts [hunk 1]
```diff
@@ -35,10 +36,6 @@ export class WelcomeComponent implements OnInit {
     });
   }
 
-  public joinMailingList(): void {
-    this.communicationService.openJoinDialog();
-  }
-
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.ts]: We need it and CommunicationService in the constructor.

File: ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html [hunk 2]
```diff
@@ -203,11 +197,8 @@ <h3 class="no-margin" translate>
                 </p>
             </div>
             <div>
-                <button (click)="joinMailingList()"
-                    [attr.aria-label]="'Common.Buttons.MailingList.AriaLabel' | translate"
-                    class="button button_medium button_secondary" translate>
-                    Common.Buttons.MailingList.Title
-                </button>
```
Comment (fat23cat) [inline ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html]: And we need this.

File: ddp-workspace/projects/toolkit/src/public-api.ts [hunk 3]
```diff
@@ -34,6 +34,6 @@ export * from './lib/components/activity/activityLink.component';
 export * from './lib/components/international-patients/internationalPatients.component';
 export * from './lib/components/session-expired/sessionExpired.component';
 export * from './lib/components/redirect-to-auth0-login/redirectToAuth0Login.component';
-export * from './lib/components/age-up/acceptAgeUpPage.component';
```
Comment (fat23cat) [inline ddp-workspace/projects/toolkit/src/public-api.ts:38]: This import should be alive.

File: ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.scss [hunk 4]
```diff
@@ -173,19 +173,12 @@
 .feeds {
     display: grid;
     grid-template-columns: 1fr 1fr;
-    grid-gap: 1rem;
```
Comment (zyme) [inline ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.scss:176]: @fat23cat did I miss any changes in this file?

Comment (fat23cat) [reply ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.scss:176]: No, you didn't. Looks good!

**Comment by fat23cat** [comment]
Can we hide texts or it's necessary requirement for widget?
image

**Comment by MocanaAtBroad** [comment]
> Can we hide texts or it's necessary requirement for widget?
> image

Took them out. c1aa4eb35eed1234e5579fdf46da9bee8e400b57

**Comment by fat23cat** [comment]
> > Can we hide texts or it's necessary requirement for widget?
> > image
> 
> Took them out. [c1aa4eb](https://github.com/broadinstitute/ddp-angular/commit/c1aa4eb35eed1234e5579fdf46da9bee8e400b57)

Looks like they are alive...

**Comment by MocanaAtBroad** [comment]
Try clearing cache. Working for me locally. (Had to use specifically https
for changes to be visible).

**Comment by fat23cat** [comment]
> Try clearing cache. Working for me locally. (Had to use specifically https
for changes to be visible).

Could you please ask someone(maybe @zyme) to test it out too?
I cleaned cache, tried to open in Edge, Firefox, Chrome, but texts are shown.

File: ddp-workspace/projects/toolkit/src/lib/components/age-up/verifyAgeUpPage.component.ts [hunk 5]
```diff
@@ -17,20 +17,20 @@ import { ToolkitConfigurationService } from '../../services/toolkitConfiguration
...
+  public handleNextUrl(url: string): void {
+    this.router.navigateByUrl(url);
+  }
 
-    public handleError(): void {
-        this.router.navigateByUrl(this.toolkitConfiguration.errorUrl);
-    }
+  public handleError(): void {
+    this.router.navigateByUrl(this.toolkitConfiguration.errorUrl);
+  }
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/toolkit/src/lib/components/age-up/verifyAgeUpPage.component.ts:35]: These file disappeared when I merged origin/develop into this branch. I have no idea why that is, so I am adding it again brute force.

File: ddp-workspace/projects/toolkit/src/lib/components/thank-you/ageUpThankYou.component.ts [hunk 6]
```diff
@@ -33,19 +33,19 @@ import { ToolkitConfigurationService } from '../../services/toolkitConfiguration
...
-        this.isEmailVerified = !!this.activatedRoute.snapshot.data.verify;
-        this.isEmailCollected = !!this.activatedRoute.snapshot.data.collect;
-        this.dashboardUrl = `/${this.toolkitConfiguration.dashboardUrl}`;
-    }
+  public ngOnInit(): void {
+    this.headerConfig.setupDefaultHeader();
+    this.isEmailVerified = !!this.activatedRoute.snapshot.data.verify;
+    this.isEmailCollected = !!this.activatedRoute.snapshot.data.collect;
+    this.dashboardUrl = `/${this.toolkitConfiguration.dashboardUrl}`;
+  }
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/toolkit/src/lib/components/thank-you/ageUpThankYou.component.ts:50]: Another one of the mysteriously disappearing files. Added again brute force.

File: ddp-workspace/projects/toolkit/src/lib/components/thank-you/lovedOneThankYou.component.ts [hunk 7]
```diff
@@ -66,22 +66,22 @@ import { HeaderConfigurationService } from '../../services/headerConfiguration.s
...
-        this.headerConfig.setupDefaultHeader();
-    }
+  public ngOnInit(): void {
+    this.phone = this.toolkitConfiguration.phone;
+    this.email = this.toolkitConfiguration.infoEmail;
+    this.phoneHref = `tel:${this.toolkitConfiguration.phone}`;
+    this.emailHref = `mailto:${this.toolkitConfiguration.infoEmail}`;
+    this.useRedesign = this.toolkitConfiguration.enableRedesign;
+    this.headerConfig.setupDefaultHeader();
+  }
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/toolkit/src/lib/components/thank-you/lovedOneThankYou.component.ts:86]: Another goner file.

**Review by fat23cat — APPROVED** [review]

**Pull request merged**